components/application-broker/internal/broker/deprovision.go

```diff
@@ -51,7 +73,7 @@ func (svc *DeprovisionService) Deprovision(ctx context.Context, osbCtx osbContex
 	case IsNotFoundError(err):
 		return nil, err
 	case err != nil:
-		return nil, errors.Wrap(err, "while checking if instance is already deprovisioned")
+		return nil, errors.Wrap(err, "checking if instance is already deprovisioned")
```

Comment:
Why do you change our error messages? we have accepted conventions of error in the whole component, it shouldn't be changed

Reply:
I will revert it. Sorry for that

Reply:
Done